__test__/deployBuilderNft.ts

```diff
@@ -7,17 +7,21 @@ import { generateWallets } from './generateWallets';
 export async function deployBuilderNftContract({ USDCContractAddress }: { USDCContractAddress?: Address } = {}) {
   const { adminAccount: admin, thirdUserAccount: proceedsReceiverAccount } = await generateWallets();
 
-  const implementation = await viem.deployContract('BuilderNFTSeasonOneImplementation01');
+  const implementation = await viem.deployContract('BuilderNFTSeasonOneImplementation01', [], {
+    client: { wallet: admin }
```

Comment:
Fixed deploys to use a specific wallet as the client, and not use default viem data